PolymerComponent.js

```diff
@@ -166,6 +174,11 @@ class PolymerComponent extends React.Component {
 
         }
 
+        // classes
+        if (props.className) {
+            this.element.setAttribute('class', props.className);
```

Comment:
This won't ever be able to be removed because 'class' only gets set when it exists. Maybe use `props.className || ''`?